client/apps/settings/redux-middleware.js

```diff
@@ -10,7 +10,7 @@ import { fetchShippingMethodSchemaSuccess } from 'woocommerce/woocommerce-servic
 import { openShippingZoneMethod } from 'woocommerce/state/ui/shipping/zones/methods/actions';
 
 const middlewareActions = {
-	[  WOOCOMMERCE_SERVICES_SHIPPING_ZONE_METHOD_SETTINGS_REQUEST_SUCCESS ]: ( { siteId, instanceId, data }, dispatch ) => {
+	[  WOOCOMMERCE_SERVICES_SHIPPING_ZONE_METHOD_SETTINGS_REQUEST_SUCCESS ]: ( { siteId, data }, dispatch ) => {
```

Comment:
I guess the new version of `eslint` is better now, it didn't catch this unused variable before.